src/pages/studyView/charts/scatterPlot/StudyViewDensityScatterPlot.tsx

```diff
@@ -22,6 +23,30 @@ import { interpolatePlasma } from 'd3-scale-chromatic';
 import { DensityPlotBin } from 'cbioportal-ts-api-client';
 import { RectangleBounds } from 'pages/studyView/StudyViewUtils';
 
+class ScaleCapt extends React.Component<any, any> {
```

Comment:
window?   
also, lets make these function components.  we gotta get on the bandwagon